PenLineModel.cpp

```diff
@@ -190,19 +181,8 @@ void PenLineModel::redo() {
 }
 
 void PenLineModel::clear() {
```

Comment:
I would also change the ```clear()``` function in penModel to ```clearUndoStack()```. clear() makes me think that it's going to ```clear()``` the penModel's data instead of clearing the undo stack.